README.md

```diff
@@ -1,6 +1,6 @@
 # Solid JavaScript Authentication - solid-client-authn
 
-`solid-client-authn` is a suite of libraries designed to authenticate with Solid identity servers.
+`solid-client-authn` is a suite of libraries designed to authenticate against Solid resource servers.
```

Comment:
Hi @RubenVerborgh  - thanks for pointing this out. I don't like the original either, but I don't think the latter quite fits or 'works' either. I chatted with @acoburn about it, and we came up with this suggestion instead:
```suggestion
`solid-client-authn` is a suite of libraries designed to manage the authentication required to access protected resources hosted by Solid servers.
```
@NSeydoux - have you any suggested improvements...?

Reply:
LGTM; perhaps `s/designed //` and `s/hosted by /on/` because it's quite verb-heavy.

Reply:
That works for me too

Reply:
Sorry - not sure how I didn't see these comments until now, but yeah, those suggestions are great, so updating to:
```suggestion
`solid-client-authn` is a suite of libraries to manage the authentication required to access protected resources on Solid servers.
```